### DISCUSSION TITLE
Is it possible to shuffle the cards?

### DISCUSSION THREAD

**Discussion by delucca** [st3v3nmw/obsidian-spaced-repetition#253] [Q&A] [answered]
Hi everyone :)

I've noticed that usually the app has an order to show me the cards. For example, if I've added 5 cards, it will show me in a given order.

It is possible to shuffle it? Mixing cards from multiple decks?

**Comment by st3v3nmw** [comment, marked as answer]
Currently, it's only possible to shuffle within a deck through the Randomize Cards Order settings option. You could open an issue for mixing cards from multiple decks.

**Reply by delucca** [reply]
Thanks, I've just opened #257
